apps/web/components/utils.tsx

```diff
@@ -269,3 +269,45 @@ export async function getAllCategories() {
     return [];
   }
 }
+
+export async function getAllMCQs() {
+  try {
+    const mcqs = await db.problem.findMany({
+      where: {
+        type: "MCQ",
+      },
+      include: {
+        mcqQuestions: true,
+      },
+    });
+    return mcqs;
+  } catch (e) {
+    return [];
+  }
+}
+
+export async function createMCQ(data: any) {
+  try {
```

Comment:
Shuold be protected, only admins should be allowed to do this

Reply:
function is called in admin page only 
or do mean anything else?